### ISSUE TITLE
A way to pass props to inner scrollable container in OverlayScrollbarsComponent for react?

### ISSUE THREAD

**Issue by Andrii-Vovk** [KingSora/OverlayScrollbars#674]
**Is your feature request related to a problem? Please describe.**
Yes. Some infinite scroll libraries require to put an id on the scrollable target, to allow them to check when to load the next page.

**Describe the solution you'd like**
Some way of passing props to the inner container, like innerProps or innerAttributes, so that an id can be set on first render.

**Describe alternatives you've considered**
Querying the dom and setting the id, but that would require to defer showing the infinite scroll component until that is done, and generally feels like a bad solution.

**Additional context**
The lib in question is react-infinite-scroll-component.

**Comment by KingSora** [comment]
Good day @Andrii-Vovk 

I consider this as "advanced" usage and therefore you should be using the `useOverlayScrollbars` hook instead of the component.
With the hook you can decide yourself how you markup should look like:

```js
const MyComponent = () => {
  const targetRef = useRef(null);
  const viewportRef= useRef(null);
  const [initialize] = useOverlayScrollbars({ options, events, defer });
  
  useEffect(() => {
    if (targetRef.current && viewportRef.current) {
      initialize({
        target: targetRef.current,
        elements: {
          viewport: viewportRef.current,
        },
      });
    }
  }, []);
  
  return (
    <div data-overlayscrollbars-initialize="" ref={targetRef}>
      <div data-overlayscrollbars-contents="" ref={viewportRef}>
        {children}
      </div>
    </div>
  );
}
```

**Comment by Andrii-Vovk** [comment]
Yep, figured that out, really sorry for not closing the issue earlier. Thank you!
